### PR TITLE
Add Rangefinder to IRIS-AP

### PR DESCRIPTION
**Pull request by mhl787156**
This PR adds a rangefinder to the IRIS-AP simulation

It is simply directly added to the vehicle xacro file

TODO
- [x] Check orientation of rangefinder

### CODE REVIEW

**Comment by mhl787156** [comment]
@rob-clarke Do you have some time to have a look at this and confirm the orientation? I'm not sure how to orient the rangefinder downwards, I'm not sure if I have attempted it in the correct place. I have tried here:

```
    <joint name="iris_hokuyo_mount" type="revolute">
      <parent>iris::base_link</parent>
      <child>hokuyo_link</child>
      <axis>
        <limit>
          <lower>0</lower>
          <upper>0</upper>
        </limit>
        <xyz>-1 0 1</xyz>
        <rpy>0 0.0 0</rpy>
        <use_parent_model_frame>true</use_parent_model_frame>
      </axis>
    </joint>
```
And changing the pitch value in rpy, but it didnt seem to make a difference to the range reading (always around 1.14)

**Comment by mhl787156** [comment]
Orientation of rangefinder has been confirmed. Min distance set to 0.01, max distance set to 20.0. The inertia has been set to negligible to avoid mass affect drone flight. The topic is outputed as a `sensor_msgs/msg/Range` on topic `vehicle_x/laser/out`. It is modeled as a single reading infrared laser rangefinder. 

Confirmed that it works and gives valid values through running the example controller. Note that there may will probably be a fixed offset between read values and the vehicles local position height due to the positioning of the vehicle frame axis. The rangefinder is attached to the vehicle base link.

File: simulator/vehicles/iris-ap/models/iris_ap/model.sdf.xacro [hunk 1]
```diff
@@ -27,6 +28,79 @@
...
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.0001</iyy>
+          <iyz>0</iyz>
+          <izz>0.0001</izz>
+        <!-- low intertia necessary to avoid to disturb the drone -->
+        </inertia>
+      </inertial>
+      <!-- Visuals / Collisions omitted for this example -->
+      <sensor type="gpu_ray" name="head_hokuyo_sensor">
```
Comment (rob-clarke) [inline simulator/vehicles/iris-ap/models/iris_ap/model.sdf.xacro:46]: Does gpu_ray work under windows?

**Review by rob-clarke — DISMISSED** [review]
Assuming `gpu_ray` works without having to do docker gpu stuff, I assume it would also work on Windows but worth checking.

File: simulator/vehicles/iris-ap/models/iris_ap/model.sdf.xacro (hunk 1 above)
Comment (mhl787156) [reply simulator/vehicles/iris-ap/models/iris_ap/model.sdf.xacro:46]: unsure, need to test. Might require changing to `ray` as its only a 1D `ray` anyway. I will just change to ray simplicity

**Review by rob-clarke — APPROVED** [review]

**Pull request merged**